packages/@lwc/babel-plugin-component/src/component.ts

```diff
@@ -68,7 +68,7 @@ export default function ({ types: t }: BabelAPI): Visitor<LwcBabelPluginPass> {
             const hasIdentifier = t.isIdentifier((node as types.ClassDeclaration).id);
             if (hasIdentifier) {
                 statementPath!.insertBefore(node);
-                node = (node as types.ClassDeclaration).id;
+                node = (node as types.ClassDeclaration).id!;
```

Comment:
`hasIdentifier` on line 69 checks whether this exists.

Reply:
Yes. Without this change, we get this error when trying to build:
```
 nx run @lwc/babel-plugin-component:build
$ rollup --config ../../../scripts/rollup/rollup.config.js
       
       /Users/jye/dev/lwc/packages/@lwc/babel-plugin-component/src/index.ts → dist/index.js, dist/index.cjs.js...
       [!] (plugin typescript) RollupError: @rollup/plugin-typescript TS2322: Type 'Identifier | null | undefined' is not assignable to type 'ClassDeclaration | FunctionDeclaration | Expression'.
         Type 'undefined' is not assignable to type 'ClassDeclaration | FunctionDeclaration | Expression'.
       src/component.ts (71:17)
       
       71                 node = (node as types.ClassDeclaration).id;
```

Reply:
Bizarre. No problem though!